templates/cassiopeia/scss/vendor/joomla-custom-elements/joomla-alert.scss

```diff
@@ -9,13 +9,17 @@
   width: 100%;
   min-width: 16rem;
   padding: 0;
-  margin-bottom: 1rem;
+  margin-bottom: 0;
   color: var(--gray-dark);
   background-color: $white;
   border: 1px solid var(--alert-accent-color, transparent);
   border-radius: .25rem;
   transition: opacity .15s linear;
 
+  + joomla-alert {
```

Comment:
@hans2103 Meanwhile I understand the "+" selector, did read a bit CSS docs ;-)
But I think this could be risky: As soon as one makes an override and adds a `<div>` between the previous element and the one with class joomla-alert, it might not fit anymore.
Is this worry right, and if so, is there a way avoid that?
Or am I completely wrong and it is just what is needed andf all is fine?

Reply:
Changing it into '+ *' solves possible issues. Will do that Monday.

Reply:
Thanks in advance.

Reply:
replacement done